### DISCUSSION TITLE
Can this be a Colab Notebook?

### DISCUSSION THREAD

**Discussion by astro9ine** [bmaltais/kohya_ss#1019] [General]
Can this be a Colab Notebook?

**Comment by shanmugad** [comment]
Kohya SS : https://colab.research.google.com/github/panguin6010/kohya_ss_google_colab/blob/master/kohya_ss_colab.ipynb

**Comment by astro9ine** [comment]
this is exactly what i was looking for, thank u so much

**Comment by DarkAlchy** [comment]
Naughty for Colab free as we aren't allowed to use WebUI now. :(
